journal/modernized/17-sermon-on-the-mount.adoc

```diff
@@ -1214,10 +1233,12 @@ As for the morrow, we need not to be too thoughtful or anxious concerning it,
 for we know not whether we shall live to enjoy it, so that as Christ says,
 "`Sufficient unto the day is the evil thereof.`"
 
-"`Judge not, that you be not judged:
+====
 
-"`For with what judgment you judge, you shall be judged; and with what measure you use,
-it shall be measured to you again.`"
+[.offset]
+_"`Judge not, that you be not judged:
+For with what judgment you judge, you shall be judged; and with what measure you use,
+it shall be measured to you again.`"_
 
 A great and wise expression, from a righteous and just Judge;
 the Judge of heaven and of earth, to whom all power in both is given;
```

Comment:
Looks like you forgot to do the offset, and the combing of the verses starting down online 1239